Wrapping/Generators/Python/itk/support/helpers.py

```diff
@@ -173,3 +176,123 @@ def image_filter_wrapper(*args, **kwargs):
+        if os.name == 'nt':
+            return 'uint32_t', 1
+        else:
+            return 'uint64_t', 1
+    mangle = None
+    pixelType = 1
+    if component == itk.SL:
+        if os.name == 'nt':
+            return 'int32_t', 1,
+        else:
+            return 'int64_t', 1,
+    if component in (itk.SC, itk.UC, itk.SS, itk.US, itk.SI, itk.UI, itk.F,
+            itk.D, itk.B):
+        mangle = component
+    elif component in [i[1] for i in itk.Vector.items()]:
+        mangle = itk.template(component)[1][0]
+        pixelType = 5
+    elif component == itk.complex[itk.F]:
+        # complex float
+        return 'float', 10
+    elif component == itk.complex[itk.D]:
+        # complex float
+        return 'double', 10
+    elif component in [i[1] for i in itk.CovariantVector.items()]:
+        # CovariantVector
+        mangle = itk.template(component)[1][0]
+        pixelType = 7
+    elif component in [i[1] for i in itk.Offset.items()]:
+        # Offset
+        return 'int64_t', 4
+    elif component in [i[1] for i in itk.FixedArray.items()]:
+        # FixedArray
+        mangle = itk.template(component)[1][0]
+        pixelType = 11
+    elif component in [i[1] for i in itk.RGBAPixel.items()]:
+        # RGBA
+        mangle = itk.template(component)[1][0]
+        pixelType = 3
+    elif component in [i[1] for i in itk.RGBPixel.items()]:
+        # RGB
+        mangle = itk.template(component)[1][0]
+        pixelType = 2
+    elif component in [i[1] for i in itk.SymmetricSecondRankTensor.items()]:
+        # SymmetricSecondRankTensor
+        mangle = itk.template(component)[1][0]
+        pixelType = 8
+    else:
+        raise RuntimeError('Unrecognized component type: {0}'.format(str(component)))
+    _python_to_js = {
+        itk.SC: 'int8_t',
+        itk.UC: 'uint8_t',
+        itk.SS: 'int16_t',
+        itk.US: 'uint16_t',
+        itk.SI: 'int32_t',
+        itk.UI: 'uint32_t',
+        itk.F: 'float',
+        itk.D: 'double',
+        itk.B: 'uint8_t'
+    }
+    imageType = dict(
+        dimension=itkimage.GetImageDimension(),
+        componentType=_python_to_js[mangle],
+        pixelType=pixelType,
+        components=itkimage.GetNumberOfComponentsPerPixel()
+    )
+    return imageType
+
+
+def image_type_from_wasm_type(jstype):
+    import itk
+
+    _pixelType_to_prefix = {
+        1: '',
+        2: 'RGB',
+        3: 'RGBA',
+        4: 'O',
+        5: 'V',
+        7: 'CV',
+        8: 'SSRT',
+        11: 'FA'
+    }
+    pixelType = jstype['pixelType']
+    dimension = jstype['dimension']
+    if pixelType == 10:
+        if jstype['componentType'] == 'float':
+            return itk.Image[itk.complex, itk.F], np.float32
+        else:
+            return itk.Image[itk.complex, itk.D], np.float64
+
+    def _long_type():
+        if os.name == 'nt':
+            return 'LL'
+        else:
+            return 'L'
+    prefix = _pixelType_to_prefix[pixelType]
+    _js_to_python = {
```

Comment:
`int64_t` is absent from `_python_to_js`. Is this on purpose?

Reply:
I'm not sure. Following Matt's suggestion [here](https://github.com/InsightSoftwareConsortium/ITK/issues/1091#issuecomment-863227855), the helper functions are basically copies of [these functions](https://github.com/InsightSoftwareConsortium/itkwidgets/blob/c4f3b158719bbb2720ef2fadcc3df8990a3feb95/itkwidgets/trait_types.py#L175-L270) from itkwidgets. Perhaps @thewtex can provide more detail? It looks like he is the original author of the `_type_to_image`/`_image_to_type` functions.

> To get the state, here is an example that represents the itk.Image state as a Python Dictionary of fundamental types + a np.ndarray (which has pickle support): https://github.com/InsightSoftwareConsortium/itkwidgets/blob/c4f3b158719bbb2720ef2fadcc3df8990a3feb95/itkwidgets/trait_types.py#L163-L172.

Reply:
Yes, these could please be added, thanks!

Reply:
Ok, I've added these to `_python_to_js`. I think they should all map to the 64 bit type because all the numpy types are 64 bit when I look at `itk.SLL.dtype` / itk.SL.dtype / etc.

```python
    _python_to_js = {
        ....
        # new code added
        itk.SL: 'int64_t',
        itk.UL: 'uint64_t',
        itk.SLL: 'int64_t',
        itk.ULL: 'uint64_t',
    }
```

Reply:
Yes, we should correct this for Windows, -- `signed long` (`SL`) and `unsigned long` (`UL`) are 32-bit on Windows :man_shrugging: .

Reply:
Thank you!